pkgs/by-name/wp/wps-fonts/package.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "wps-fonts";
+  version = "2017-08-16";
```

```suggestion
  version = "0-unstable-2017-08-16";
```

https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming